### PR TITLE
WINDUP-1989: Enhance Documentation for Maven Plugin

### PR DESCRIPTION
**Pull request by huffmanca**
* Included section on multi-module projects.
* Moved the `Review the Reports` section out, and adjusted the links to point to the CLI guides.
* Updated a few attribute descriptions in the reference section

### CODE REVIEW

**Comment by huffmanca** [comment]
@jsight,

This is the latest version of the Maven Plugin guide. Would it be possible to review the current content for any issues or suggestions?

File: docs/topics/maven-arguments.adoc [hunk 1]
```diff
@@ -14,11 +14,11 @@ The following is a detailed description of the available {ProductShortName} {Mav
 |explodedApps |Flag to indicate that the provided input directory contains source files for a single application. See the xref:input_file_type_arguments[Input File Argument Tables] for details.
 |exportCSV |Flag to export the report data to a CSV file on your local file system. {ProductShortName} creates the file in the directory specified by the `outputDirectory` argument. The CSV file can be imported into a spreadsheet program for data manipulation and analysis. For details, see xref:export_csv[Export the Report in CSV Format].
 |includeTags |A list of tags to use. When specified, only rules with these tags will be processed. 
-|inputDirectory |Specify the path to the directory containing the applications to be analyzed. This argument defaults to the project's source directory. See xref:input_argument[Specify the Input Directory] for more information.
+|inputDirectory |Specify the path to the directory containing the applications to be analyzed. This argument defaults to `{project.basedir}/src/main/`. See xref:input_argument[Specify the Input Directory] for more information.
```
Comment (jsight) [inline docs/topics/maven-arguments.adoc]: I think this section needs documentation for the new "windupHome" parameter as well.

File: docs/topics/maven-multi-module.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
+= Run the {MavenNameTitle} with Multiple Modules
+
+To use the {MavenName} in a project with multiple modules, place the configuration inside the parent's `pom.xml`. During execution the {MavenName} will generate a single report that contains the analysis for the parent and any child modules.
+
+NOTE: It is strongly recommended to set `inherited` to false in multi-module projects; otherwise, the {MavenName} will be executed when each child is compiled, resulting in drastically increased run times.
```
Comment (jsight) [inline docs/topics/maven-multi-module.adoc]: I might say that it also runs it multiple times when the parent module is compiled.

**Comment by jsight** [comment]
Just a few comments. It looks good overall, though.

File: docs/topics/maven-arguments.adoc (hunk 1 above)
Comment (huffmanca) [reply docs/topics/maven-arguments.adoc]: This parameter has now been included.

**Comment by huffmanca** [comment]
@jsight,

The feedbacks have been addressed. Let me know if this looks good to merge now, and thank you for the review!

**Pull request merged**